tools/create_callback_client_certificate.sh

```diff
@@ -28,8 +32,19 @@ certC=${input:-${certC}}
 #        * ) echo "Please answer Y(es) or N(o).";;
 # esac
 #done
+
+case "$(uname -s)" in
+  MINGW32*|MSYS*|MINGW*)
+    # Work around MinGW/MSYS's path conversion (http://www.mingw.org/wiki/Posix_path_conversion) 
+    ertSubject="//C=${certC}\CN=${certCN}\O=EFGS DEV Org"
```

Comment:
```suggestion
    certSubject="//C=${certC}\CN=${certCN}\O=EFGS DEV Org"
```

typo

Reply:
Oh we missed that one! Thanks for the catch.